docs/core/testing/unit-testing-best-practices.md

```diff
@@ -3,56 +3,57 @@ title: Best practices for writing unit tests
 description: Learn best practices for writing unit tests that drive code quality and resilience for .NET Core and .NET Standard projects.
 author: jpreese
 ms.author: wiwagn
-ms.date: 07/28/2018
+ms.date: 07/06/2022
```

Comment:
This is the incorrect date format. It should be `MM/DD/YYYY`.
```suggestion
ms.date: 06/07/2022
```

Reply:
Ok, updated accordingly. Thanks!